packages/jaeger-ui/src/components/QualityMetrics/BannerText.test.js

```diff
@@ -13,32 +13,38 @@
 // limitations under the License.
 
 import React from 'react';
-import { shallow } from 'enzyme';
-
+import { render } from '@testing-library/react';
+import '@testing-library/jest-dom';
 import BannerText from './BannerText';
 
 describe('BannerText', () => {
   it('renders null when props.bannerText is falsy', () => {
-    expect(shallow(<BannerText />).type()).toBe(null);
+    const { container } = render(<BannerText />);
+    expect(container.firstChild).toBe(null);
   });
 
   it('renders header when props.bannerText is a string', () => {
-    expect(shallow(<BannerText bannerText="foo text" />)).toMatchSnapshot();
+    const { getByText } = render(<BannerText bannerText="foo text" />);
```

Comment:
You can use most of these functions using the `screen` object from `@testing-library/react`.
Example,

```js
render(<BannerText bannerText="foo text" />);
expect(screen.getByText('foo text')).toBeInTheDocument();
```

Same goes for other such usages as well